stories/modules-tagmanager-setup.stories.js

```diff
@@ -256,59 +256,6 @@ storiesOf( 'Tag Manager Module/Setup', module )
 			padding: 0,
 		}
 	)
-	.add(
```

Comment:
Same, here - let's keep one of these _Existing tag..._ stories and remove the access stuff from it.